db/db_bloom_filter_test.cc

```diff
@@ -2244,6 +2244,48 @@ TEST_F(DBBloomFilterTest, MemtableWholeKeyBloomFilterMultiGet) {
 
   db_->ReleaseSnapshot(snapshot);
 }
+
+TEST_F(DBBloomFilterTest, TestMemtableBloomAndWBM) {
+  Options options = CurrentOptions();
+  options.arena_block_size = 4096;
+  std::shared_ptr<Cache> cache = NewLRUCache(LRUCacheOptions(
+      10000000 /* capacity */, 1 /* num_shard_bits */,
+      false /* strict_capacity_limit */, 0.0 /* high_pri_pool_ratio */,
+      nullptr /* memory_allocator */, kDefaultToAdaptiveMutex,
+      kDontChargeCacheMetadata));
+
+  options.write_buffer_size = 4000000;
+  options.write_buffer_manager.reset(new WriteBufferManager(4000000, cache));
+  Reopen(options);
+  ASSERT_OK(Put("foo", "bar"));
+
+  // Just the start of a memtable, no Bloom, rounded up to kSizeDummyEntry
+  ASSERT_GE(cache->GetUsage(), 1024U);
+  ASSERT_LE(cache->GetUsage(), 256U * 1024U);
```

Comment:
nit: use static constexpr std::size_t kSizeDummyEntry = CacheReservationManagerImpl<CacheEntryRole::kMisc>::GetDummyEntrySize(); ? Same for upper bound below just so it's easier to sea how many kSizeDummyEntry in the upper bound